src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -51,7 +52,7 @@
 from psyclone.psyGen import PSyFactory
 from psyclone.errors import InternalError, GenerationError
 from psyclone.psyir.symbols import (
-    DataSymbol, ContainerSymbol, SymbolTable, RoutineSymbol,
+    DataSymbol, ContainerSymbol, SymbolTable, RoutineSymbol, DataTypeSymbol,
```

Comment:
Pylint says the DataTypeSymbol is unused

Reply:
and it is correct.